src/metadata.json

```
@@ -3,6 +3,6 @@
    "name": "Notification Banner Position",
    "description": "Changes position of the notification banner from the default to the right side of the screen.",
    "version": 4,
```

Please bump the Gnome extension versions. This is helpful when uploading to their website.
```suggestion
   "version": 5,
```

---

Actually, it also needs an entry in the CHANGELOG.md. I can do that for you and merge this branch, don't worry.

---

hey mate, I added both of the changes

---

Thanks, mate.

Gonna add some info to the README about how to do this upgrade, but also gonna try to automate this everytime a new Gnome version is out, since this code is simple enough that it probably won't break from one version to another.